Eastern-Research-Group/mywaterway
# Feature/11 native monitoring pages

## Related Issues:
* [HMW-11](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-11)

## Main Changes:
* Refactored the JSX on the Monitoring Report page to eliminate nested ternaries.
* Fixed a bug in the download section where an indeterminate checkbox could cause a parent to be checked.

## Steps To Test:
1. Go to a station page, e.g. [http://localhost:3000/station/STORET/31ORWUNT/31ORWUNT-OR502.2M/](http://localhost:3000/station/STORET/31ORWUNT/31ORWUNT-OR502.2M/)
2. In the _Download Station Data_ section, deselect "Metals"
3. Expand "Metals"
4. Expand "Inorganics, Major, Metals"
5. Select "Calcium"
6. Observe that the checkbox for "Metals" is indeterminate. Previously, with an indeterminate value of 2, it would have been checked, since Metals has 2 children.
